gateway-spi/src/main/java/org/apache/knox/gateway/topology/monitor/RemoteConfigurationMonitor.java

```diff
@@ -16,14 +16,11 @@
  */
 package org.apache.knox.gateway.topology.monitor;
 
-import org.apache.knox.gateway.services.config.client.RemoteConfigurationRegistryClient;
-
-public interface RemoteConfigurationMonitor {
-
-    void start() throws Exception;
-
-    void stop() throws Exception;
-
-    RemoteConfigurationRegistryClient getClient();
+import org.apache.knox.gateway.services.Service;
 
+public interface RemoteConfigurationMonitor extends Service {
```

Comment:
This should be called a service to align with current naming conventions: `RemoteConfigurationMonitorService`